server/inspect.go

```diff
@@ -170,6 +172,42 @@ func (s *Server) GetInfoMux(enableProfile bool) *bone.Mux {
 		}
 	}))
 
+	mux.Get(InspectPauseEndpoint+"/:id", http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		containerID := bone.GetValue(req, "id")
+		ctr := s.GetContainer(containerID)
+
+		if ctr == nil {
+			http.Error(w, fmt.Sprintf("can't find the container with id %s", containerID), http.StatusNotFound)
+			return
+		}
+		if err := s.PauseContainer(s.stream.ctx, ctr); err != nil {
```

Comment:
instead of going through ContainerServer, you can just call `s.Runtime().PauseContainer` here. Same below with UnpauseContainer